client/state/data-layer/wpcom/plans-request.js

```diff
@@ -0,0 +1,27 @@
+import wpcom from 'lib/wp';
```

Comment:
It's more like having one file for each type of data request we want. I imagine this layer as the translation from the internal Calypso representation of data to the server's representation of data.

Imagine that we know for some reason that we _always_ want to update the related site object when we request site settings. We could expose `SITE_SETTINGS_REQUEST` in the `sites-request` file so that it watches for that action and dispatches the fetch accordingly.

Not 💯 on this yet and I think some of that needs to come after using the pattern and exploring it. The main goal here is to make navigating the code easy and to make reverse-lookup easy (we issues a request to this endpoint, which file should I look for?). To that end, maybe one per endpoint is ideal.

Thoughts?